target/linux/realtek/files-5.15/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -537,22 +536,22 @@ static int rtl8226_config_aneg(struct phy_device *phydev)
 		if (ret)
 			goto out;
 		/* AutoNegotiationEnable */
-		v = phy_read_mmd(phydev, MMD_AN, 0);
+		v = phy_read_mmd(phydev, MDIO_MMD_AN, 0);
 		if (v < 0)
 			goto out;
 
-		v |= BIT(12); /* Enable AN */
-		ret = phy_write_mmd(phydev, MMD_AN, 0, v);
+		v |= BMCR_ANENABLE;
```

Comment:
Although it has the same value, it is preferred to use `MDIO_AN_CTRL1_ENABLE` for MMDs.